core/src/main/java/com/hcl/ecomm/core/services/impl/LoginServiceImpl.java

```diff
@@ -0,0 +1,88 @@
+package com.hcl.ecomm.core.services.impl;
+
+import com.hcl.ecomm.core.config.LoginServiceConfig;
+import com.hcl.ecomm.core.services.LoginService;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.propertytypes.ServiceDescription;
+import org.osgi.service.component.propertytypes.ServiceRanking;
+import org.osgi.service.component.propertytypes.ServiceVendor;
+import org.osgi.service.metatype.annotations.Designate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(service = { LoginService.class })
+@Designate(ocd = LoginServiceConfig.class)
+
+@ServiceDescription("Login service")
+@ServiceRanking(1001)
+@ServiceVendor("Nithesh")
+
+public class LoginServiceImpl implements LoginService {
+
+	private static final Logger LOG = LoggerFactory.getLogger(LoginServiceImpl.class);
+
+	@Activate
+	private LoginServiceConfig config;
+
+	@Override
+	public String getDomainName() {
+		return config.loginservice_domainName_string();
+	}
+
+	@Override
+	public String getServicePath() {
+		return config.loginservice_servicePath_string();
+	}
+
+	@Override
+	public String getUsername() {
+		return config.loginservice_username_string();
+	}
+
+	@Override
+	public String getPassword() {
+		return config.loginservice_password_string();
+	}
+
+	@Override
+	public String getToken() {
+
+		String scheme = "http";
```

Comment:
@nitheshkumar-hcl : what in case scheme is https

Reply:
for connecting to magento using rest api we need to provide schema as http,as it is not secure.if we use schema as https  we will not be able to connect to magento